packages/twenty-front/src/modules/object-record/record-show/record-detail-section/components/RecordDetailRelationSection.tsx

```diff
@@ -177,7 +179,7 @@ export const RecordDetailRelationSection = ({
               onClose={handleCloseRelationPickerDropdown}
               clickableComponent={
                 <LightIconButton
-                  className="displayOnHover"
+                  className={isMobile ? '' : 'displayOnHover'}
```

Comment:
Is this line needed?

Reply:
It doesn’t really make a difference; I just added it for code clarity.

Reply:
Okay, I think we can leave it as it was then :)

Reply:
Done!